api/client/javascript/scripts/add-as-const.ts

```diff
@@ -1,71 +1,82 @@
-import path from 'node:path'
-import { fileURLToPath } from 'node:url'
-import { type Expression, Project, SyntaxKind } from 'ts-morph'
-import type * as ts from 'typescript'
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { type Expression, Project, SyntaxKind } from "ts-morph";
+import type * as ts from "typescript";
 
 /**
  * Post-generation TypeScript transformer to add "as const" assertions to make sure the generated Zod schemas are valid
  * This fixes TypeScript errors with Zod schemas that expect literal types
  */
 
-const __filename = fileURLToPath(import.meta.url)
-const __dirname = path.dirname(__filename)
-const ZOD_FILE_PATH = path.join(__dirname, '../src/zod/index.ts')
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const ZOD_FILE_PATH = path.join(__dirname, "../src/zod/index.ts");
 
 function addAsConstAssertions() {
   const project = new Project({
-    tsConfigFilePath: path.join(__dirname, '../tsconfig.json'),
-  })
+    tsConfigFilePath: path.join(__dirname, "../tsconfig.json"),
+  });
 
-  const sourceFile = project.addSourceFileAtPath(ZOD_FILE_PATH)
-  let transformationCount = 0
+  const sourceFile = project.addSourceFileAtPath(ZOD_FILE_PATH);
+  let transformationCount = 0;
 
   // Find all variable declarations
-  const variableStatements = sourceFile.getVariableStatements()
+  const variableStatements = sourceFile.getVariableStatements();
 
   for (const statement of variableStatements) {
     // Only process exported const declarations
     if (!statement.isExported()) {
-      continue
+      continue;
     }
 
-    const declarations = statement.getDeclarations()
+    const declarations = statement.getDeclarations();
 
     for (const declaration of declarations) {
       if (declaration.getKind() !== SyntaxKind.VariableDeclaration) {
-        continue
+        continue;
       }
 
-      const name = declaration.getName()
-      const initializer = declaration.getInitializer()
+      const name = declaration.getName();
+      const initializer = declaration.getInitializer();
 
       if (!initializer) {
-        continue
+        continue;
       }
 
+      // // Special-case: union default for BillingWorkflowCollectionAlignment is missing in OAPI and becomes {}
+      // // Patch those constants to a concrete default literal expected by Zod and consumers.
+      // if (
+      //   name === 'createBillingProfileBodyWorkflowCollectionAlignmentDefault' ||
+      //   name === 'updateBillingProfileBodyWorkflowCollectionAlignmentDefault'
+      // ) {
+      //   declaration.setInitializer("{ type: 'subscription' } as const")
+      //   transformationCount++
+      //   continue
+      // }
+
```

Comment:
As far as I see this is just a formatting change as this is commented out. Can you make the formatting a seperate PR if you want that to be merged?